PHC-Android/PHC/src/main/java/phc/android/Checkin/SearchResultsFragment.java

```diff
@@ -274,129 +338,55 @@ public void onErrorResponse(VolleyError volleyError) {
-                    } catch (JSONException e2) {
-                        Log.e("Search Result JSON Error", e2.toString());
-                    } finally {
-                        editor.putBoolean("Searched", true);
-                        editor.commit();
-                    }
-                    // If we successfully load a user, we change the state to returning user
-                    mParent.setCurrentState(CheckinActivity.RegistrationState.RETURNING_USER);
-
-                    PersonalInfoFragment newFragment = new PersonalInfoFragment();
-                    FragmentTransaction transaction = getFragmentManager().beginTransaction();
-                    transaction.replace(R.id.checkin_fragment_container, newFragment, getResources().getString(R.string.sidebar_personal_info));
-                    transaction.addToBackStack(null);
-                    transaction.commit();
-
-                }
-
-                @Override
-                public void onError(Exception exception) {
-                    Log.e("Search Response Error", exception.toString());
-                }
-            };
-            sendRequest(request, callback);
-        } catch (Exception e) {
-            Log.e("Search Select Error", e.toString());
-        }
-
-    }
-
     /**
-     * Helper that sends request to server and print result in text field.
-     *
-     * @param request - The request object that gets executed by the SF SDK
-     * @param callback - The functions that get called when yhe response comes back
-     *                   Modify UI elements here.
+     * Response Listener for user info request
      */
-    private void sendRequest(RestRequest request, AsyncRequestCallback callback) {
-
-        try {
+    private class UserInfoResponseListener implements Response.Listener<JSONObject> {
 
-            ((CheckinActivity) getActivity()).client.sendAsync(request, callback);
-
-        } catch (Exception error) {
-            Log.e("SF Request", error.toString());
+        @Override
+        public void onResponse(JSONObject jsonObject) {
+            SharedPreferences sharedPreferences = getActivity().getSharedPreferences(SEARCH_RESULT, 0);
+            SharedPreferences.Editor editor = sharedPreferences.edit();
+            try {
+                editor.putString("SS_Num", jsonObject.getString("SS_Num__c"));
+                editor.putString("FirstName", jsonObject.getString("FirstName"));
+                editor.putString("LastName", jsonObject.getString("LastName"));
+                editor.putString("Phone", jsonObject.getString("Phone"));
+                editor.putString("Birthdate", jsonObject.getString("Birthdate__c"));
+                editor.putString("Email", jsonObject.getString("PersonEmail"));
+                editor.putString("Gender", jsonObject.getString("Gender__c"));
+                editor.putString("Language", jsonObject.getString("Primary_Language__c"));
```

Comment:
you'll also need to include Ethnicity Spinner, and the LGBT, Foster Care, and Veteran checkboxes